integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonTableCompactor.scala

```diff
@@ -90,7 +90,6 @@ class CarbonTableCompactor(carbonLoadModel: CarbonLoadModel,
 
     while (loadsToMerge.size() > 1 || needSortSingleSegment(loadsToMerge)) {
       val lastSegment = sortedSegments.get(sortedSegments.size() - 1)
-      deletePartialLoadsInCompaction()
```

Comment:
if you are removing this, how are you handling the case of segment number same in next compaction retry as we dont have any entry for table status file during compaction.

Reply:
In this case, clean files have no chance to clean data.
keep the stale data in the folder, it will not impact compaction.

Reply:
but the stale data will always be present inside segment folder right? by chance assume if the segment file is corrupted or deleted, still carbon should pass the query, which it does by listing, in that case we will get the wrong data or query will always fail. I think we need to have some way to clean them, what you think @QiangCai @ajantha-bhat

Reply:
I think this will go as a stale segment in trash folder when clean files is called right ?